src/components/Input/CheckBox/CheckBoxProps.ts

```diff
@@ -0,0 +1,14 @@
+type SizeType = 'small' | 'medium' | 'large' | undefined;
+type CheckBoxType = 'primary' | 'secondary' | undefined;
```

Comment:
```suggestion
type SizeType = 'small' | 'medium' | 'large';
type CheckBoxType = 'primary' | 'secondary';
```

gaperlu | undefined lagi.. tinggal di interfacenya tambahin tanda tanya